lib/assets/stylesheets/_govuk_tech_docs.scss

```diff
@@ -1,4 +1,15 @@
-$govuk-assets-path: "/assets/govuk/assets/" !default;
+@function tech-docs-image-url-function($filename) {
+  @return image-url($govuk-images-path + $filename)
+}
+
+@function tech-docs-font-url-function($filename) {
+  @return font-url($govuk-fonts-path + $filename)
+}
+
+$govuk-font-url-function: 'tech-docs-font-url-function';
+$govuk-image-url-function: 'tech-docs-image-url-function';
+
+$govuk-assets-path: "/assets/govuk/assets/";
```

Comment:
I suspect this does work as intended, because presumably `$govuk-fonts-path` isn't evaluated until the function is called, but it might make people's heads hurt less if this variable was set before the functions were defined

(I appreciate that `$govuk-images-path` and `$govuk-fonts-path` haven't been defined at all at this point!)

Reply:
Hmm, yes, I don't know, I kind of thought that having it after would help people realise that `$govuk-{images,fonts}-path` are not defined here and that they need to dig deeper to understand it better. Maybe a comment would be more kind though 😂

Reply:
I've added some comments, let me know if you think more detail is needed.